pkg/certificate/providers/vault/tools.go

```diff
@@ -11,12 +11,8 @@ func getDurationInMinutes(validityPeriod time.Duration) string {
 	return fmt.Sprintf("%dh", validityPeriod/time.Hour)
 }
 
-func getIssueURL(role vaultRole) vaultPath {
-	return vaultPath(fmt.Sprintf("pki/issue/%+v", role))
-}
-
-func getRoleConfigURL(role vaultRole) vaultPath {
```

Comment:
note: this is valid to remove since it is never used. We can delete the corresponding test as well